packages/eslint-plugin/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@readme/eslint-plugin",
+  "version": "1.0.0",
+  "description": "ESLint plugin providing custom rules for ReadMe's coding standards",
+  "main": "index.js",
+  "author": "Jon Ursenbach <jon@ursenba.ch>",
+  "license": "ISC",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/readmeio/standards.git",
+    "directory": "packages/eslint-plugin"
+  },
+  "engines": {
+    "node": ">=18"
+  },
+  "scripts": {
+    "prettier": "prettier --list-different --write \"./**/**.js\"",
```

Comment:
we run `prettier` checks from the root of the monorepo so you shouldn't need this

```suggestion
```